bpfman-operator/Makefile

```
@@ -181,10 +181,7 @@ ifeq ($(VERIFY_CODEGEN), true)
 VERIFY_FLAG=--verify-only
 endif
 
-OUTPUT_PKG ?= $(shell pwd)/bpfman-operator/pkg/client
-APIS_PKG ?= $(shell pwd)/bpfman-operator
-CLIENTSET_NAME ?= versioned
-CLIENTSET_PKG_NAME ?= clientset
+PKG ?= github.com/bpfman/bpfman/bpfman-operator
 COMMON_FLAGS ?= ${VERIFY_FLAG} --go-header-file $(shell pwd)/hack/boilerplate.go.txt
```

```suggestion
COMMON_FLAGS ?= ${VERIFY_FLAG} --go-header-file ${PKG}hack/boilerplate.go.txt
```
Does this need to change too? My output from sample command (see `--go-header-file`):
```
/home/bmcfall/go/src/github.com/bpfman/bpfman/bpfman-operator/bin/client-gen \
	--clientset-name "versioned" \
	--input-base "" \
	--input "github.com/bpfman/bpfman/bpfman-operator/apis/v1alpha1" \
	--output-package "github.com/bpfman/bpfman/bpfman-operator/pkg/client/clientset" \
	 --go-header-file /home/bmcfall/go/src/github.com/bpfman/bpfman/bpfman-operator/hack/boilerplate.go.txt
```

---

Never-mind, it fails to build with that change.